guides/building.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: doc
+sublayout: guide
+title: Building Dockerfiles
+---
```

Comment:
Oh looks like you made a doc for triggers. I think I'll just expand on this to show off the SSH public keys and write a guide for submodule permissions on GH.